dataproc_jupyter_plugin/tests/test_bigquery.py

```diff
@@ -183,6 +183,7 @@ async def mock_config(config_field):
             "system": mock_system,
             "type": mock_type,
         },
+        method="POST",
```

Comment:
You need to add `allow_nonstandard_methods=True,` here in order to get the test to pass.